### PR TITLE
Add continuous integration

### LINKED ISSUE #6: Add continuous integration

### PR DESCRIPTION
**Pull request by ArwynFr**
Fixes #6

### CODE REVIEW

File: FORM_Tembelan.Tembelan/mission.sqm [hunk 1]
```diff
@@ -331,173 +332,179 @@ class AddonsMetaData
 			url="http://ace3mod.com/";
 		};
 		class Item31
+		{
+			className="AI_Driving_Control";
+			name="AI Driving Control";
+			author="RCA3";
+		};
```
Comment (ImplicitDeny) [inline FORM_Tembelan.Tembelan/mission.sqm:339]: May I ask why are we adding this new dependency ? I had a quick look to its demo video, I have to say that I'm not convinced at all...

Comment (ArwynFr) [reply FORM_Tembelan.Tembelan/mission.sqm:339]: I suggest you compare one hour of Liberation (on Radon) with and without it ...

Comment (ImplicitDeny) [reply FORM_Tembelan.Tembelan/mission.sqm:339]: Is it serverside ? According to the Armaholic page of the mod, reupload is not allowed.

Comment (ImplicitDeny) [reply FORM_Tembelan.Tembelan/mission.sqm:339]: Nevermind, it already has been added to the modpack, so I assume that any question related to reuploading the mod has already been solved.

**Pull request merged**